### ISSUE TITLE
Analytics review of floating promo box

### ISSUE THREAD

**Issue by hoosteeno** [mozilla/bedrock#9888]
## Description

Let's review the performance of the floating promo box in in #9612 to see if it is having a positive impact. If it is, let's figure out how to move forward with it. 

---

#### Success Criteria
- [ ] Analysis is attached here
- [ ] Decision made; cards built if needed

**Comment by hoosteeno** [comment]
@maxxcrawford can you tell Raphael when the card launched, and where, and anything else he needs to know?

**Comment by maxxcrawford** [comment]
@rraue Here's all the info I have on the promo box to date. 

## Timeline

- Nov 19, 2020 – Sticky Promo Box lands on /new pages (#9406)
  - Note that the `download-location` attr was set incorrectly to `other` and is visible to only non-mobile browser traffic. 
- Dec 4, 2020 – Sticky Promo box `download-location` attr revised to `sticky-promo-box` (#9721)
- Jan 21, 2020 – Sticky Promo [lands](https://github.com/mozilla/bedrock/releases/tag/2021-01-21) of 24 additional pages (https://github.com/mozilla/bedrock/pull/9780)
  - Note that these new pages ONLY apply to non-mobile, non-Firefox browser traffic. 

## Notes 
As of this comment, the only two promotions/CTAs for this element are: 
- Non Firefox: Download Firefox
- Firefox: Check out our other products (browsers, Monitor, Lockwise, Pocket and VPN) 

See [this sample report](https://analytics.google.com/analytics/web/?pli=1#/report/content-landing-pages/a36116321w64087799p65789850/_u.date00=20201101&_u.date01=20210202&explorer-table.secSegmentId=analytics.customDimension15&explorer-table.plotKeys=%5B%5B%22~2Fen-US~2Ffirefox~2F%22,%22nav%22%5D,%5B%22~2Fen-US~2Ffirefox~2F%22,%22sticky-promo-box%22%5D,%5B%22~2Fen-US~2Ffirefox~2F%22,%22primary%20cta%22%5D,%5B%22~2Fen-US~2Ffirefox~2F%22,%22other%22%5D%5D&explorer-table.advFilter=%5B%5B0,%22analytics.landingPagePath%22,%22RE%22,%22en-US%5C%5C~2Ffirefox%5C%5C~2F$%22,0%5D%5D&explorer-graphOptions.primaryConcept=analytics.goal2Completions/) generated by @hoosteeno that shows how this converts for /new pages. 


## URLs

Page list: 
- Nov 19 Deploy
	- [/firefox/](https://www.mozilla.org/firefox/)
	- [/firefox/new/](https://www.mozilla.org/firefox/new/)
	- [/firefox/browsers/](https://www.mozilla.org/firefox/browsers/)
- Jan 21 Deploy
	- [/firefox/releasenotes](https://www.mozilla.org/firefox/releasenotes)
	- [/firefox/features/adblocker/](https://www.mozilla.org/firefox/features/adblocker/)
	- [/firefox/features/bookmarks](https://www.mozilla.org/firefox/features/bookmarks)
	- [/firefox/features/fast](https://www.mozilla.org/firefox/features/fast)
	- [/firefox/features/independant](https://www.mozilla.org/firefox/features/independant)
	- [/firefox/features/memory](https://www.mozilla.org/firefox/features/memory)
	- [/firefox/features/password-manager](https://www.mozilla.org/firefox/features/password-manager)
	- [/firefox/features/private-browsing](https://www.mozilla.org/firefox/features/private-browsing)
	- [/firefox/features/safe-browser](https://www.mozilla.org/firefox/features/safe-browser)
	- [/firefox/browsers/best-browser](https://www.mozilla.org/firefox/browsers/best-browser)
	- [/firefox/browsers/browser-history](https://www.mozilla.org/firefox/browsers/browser-history)
	- [/firefox/browsers/incognito-browser](https://www.mozilla.org/firefox/browsers/incognito-browser)
	- [/firefox/browsers/what-is-a-browser](https://www.mozilla.org/firefox/browsers/what-is-a-browser)
	- [/firefox/browsers/windows-64-bit/](https://www.mozilla.org/firefox/browsers/windows-64-bit/)
	- [/firefox/browsers/compare/brave](https://www.mozilla.org/firefox/browsers/compare/brave)
	- [/firefox/browsers/compare/opera](https://www.mozilla.org/firefox/browsers/compare/opera)
	- [/firefox/browsers/compare/chrome](https://www.mozilla.org/firefox/browsers/compare/chrome)
	- [/firefox/browsers/compare/ie](https://www.mozilla.org/firefox/browsers/compare/ie)
	- [/firefox/browsers/compare/edge](https://www.mozilla.org/firefox/browsers/compare/edge)
	- [/firefox/browsers/compare/safari](https://www.mozilla.org/firefox/browsers/compare/safari)
	- [/firefox/features/](https://www.mozilla.org/firefox/features/)
	- [/firefox/browsers/quantum/](https://www.mozilla.org/firefox/browsers/quantum/)

## Screenshots: 

Download: 
<img width="347" alt="image" src="https://user-images.githubusercontent.com/2692333/106632864-d4a92080-6543-11eb-9f99-83ab22406638.png">


Other products: 
<img width="350" alt="image" src="https://user-images.githubusercontent.com/2692333/106632846-ceb33f80-6543-11eb-8371-64cd94a2463f.png">

**Comment by rraue** [comment]
The results for the newer pages are still inconclusive, as right now we do not have enough session data to see if the sticky promo box had a significant influence on the download rate. We should wait minimum 2 more weeks (better 4) to be sure on what kind of pages the sticky promo box worked and where not.

On /new, /firefox and /browsers we have enough data and can declare a significant uplift of the conversion rate. 🥳
The uplift was not all the same on these three pages, strongest was on /new, weaker one on /firefox and a small uplift on /browsers. 

Still a great success but to determine if we should expand the sticky promo to more pages, we need more data from the smaller pages, as right now, some show even a negative influence, but we need more data to determine if that sticks.

The full dataset: https://docs.google.com/spreadsheets/d/1lDAtC7CIAhISOpLA7fOZQd-scKnw9Z2R7A3yhlD0ap4/ (mozilla access only)

**Comment by rraue** [comment]
Blocked by sample size, we need to get back and redo the analysis in march again.

**Comment by maxxcrawford** [comment]
Is there any take aways from the first the pages on conversion for non-Firefox vs Firefox traffic? Specifically, does a non-download CTA work well in the floating box?

**Comment by rraue** [comment]
I did the analysis only on non-fx sessions for download rate. If we want to know what the sticky promo does for firefox users I am not sure if i am able to do that as our non-download goals are a bit clunky and I wouldn't know what to call a "conversion" in theses cases. If we want to do that, we would have to sit together with @hoosteeno and discuss

@maxxcrawford Could you elaborate in the question if other locales have the promo box (I saw it on de), which locales and when have they been launched (same as in en-US)?

**Comment by slightlyoffbeat** [comment]
This is no longer blocked, but @rraue will need to find the time to complete this task.

**Comment by rraue** [comment]
So with more time and data I looked at this again and I see some mixed messages in the data. The longer the time range I look at the harder it is of course to determine the sticky promo as the changing factor. And thats why I think the data is inconclusive: https://docs.google.com/spreadsheets/d/1wvwp9rCV5PZpX9z_YlDWUbM-9Dok2cgThMYnzCQeHwk/edit?usp=sharing

On some pages download rate went up, on some down. If we want to know if this is a good navigational and conversion tool, we should a/b test it.

Closing this issue.
